tests/Fake/FakeDomain.php

```diff
@@ -11,7 +11,7 @@ class FakeDomain implements DomainInterface
     public function __invoke(array $input)
     {
         return (new Payload())
-            ->withStatus(Payload::OK)
```

Comment:
A separate PR needs to be made against `equip/adr` to drop the deprecated status codes.